Sources/HomomorphicEncryptionProtobuf/HomomorphicEncryptionProtobuf.docc/HomomorphicEncryptionProtobuf.md

```diff
@@ -1,6 +1,6 @@
 # ``HomomorphicEncryptionProtobuf``
 
-Protocol buffer support for [HomomorphicEncryption](https://swiftpackageindex.com/apple/swift-homomorphic-encryption/main/documentation/homomorphicencryption)
+Protocol buffer support for ``HomomorphicEncryption``
```

Comment:
Fixes `Link in document summary will not be displayed` (that's true for Xcode. But it does render in SPI (e.g. https://swiftpackageindex.com/apple/swift-homomorphic-encryption/1.0.2/documentation/homomorphicencryptionprotobuf -- the summary "Protocol buffer support for [PrivateInformationRetrieval](https://swiftpackageindex.com/apple/swift-homomorphic-encryption/main/documentation/privateinformationretrieval)" has a hyperlink just fine)